tests/boot/grub_test_snapshot.pm

```diff
@@ -14,6 +14,10 @@ use testapi;
 
 sub run() {
     my $self = shift;
+
+    assert_screen "inst-bootmenu";
+    sleep 2;
```

Comment:
i pasted and copy from boot_into_console.pm test. so we have there the sleep originally